widgets/post-carousel/post-carousel.php

```diff
@@ -16,17 +16,23 @@ function sow_carousel_register_image_sizes(){
 
 function sow_carousel_get_next_posts_page() {
 	if ( empty( $_REQUEST['_widgets_nonce'] ) || !wp_verify_nonce( $_REQUEST['_widgets_nonce'], 'widgets_action' ) ) return;
-	$query = wp_parse_args(
-		siteorigin_widget_post_selector_process_query($_GET['query']),
-		array(
-			'post_status' => 'publish',
-			'posts_per_page' => 10,
-			'paged' => empty( $_GET['paged'] ) ? 1 : $_GET['paged']
-		)
-	);
-
-	$posts = new WP_Query($query);
+
+	$instance_hash = $_GET['instance_hash'];
```

Comment:
Always slightly safer to check if `$_GET['instance_hash']` isn't `empty` before accessing it to make sure we don't trigger any warnings. Can assign to `false` if it is empty and make the check on line 23 a standard boolean check.